### PR TITLE
Support AndroidStudio

### PR DESCRIPTION
**Pull request by henteko**

### CODE REVIEW

File: import-summary.txt [hunk 1]
```diff
@@ -0,0 +1,107 @@
+ECLIPSE ANDROID PROJECT IMPORT SUMMARY
```
Comment (tnj) [inline import-summary.txt]: May be we should remove this file?

**Comment by tnj** [comment]
Also we can remove `app/libs/deploygatesdk.jar` which is provided by Gradle dependencies.

**Comment by henteko** [comment]
@tnj 
fix!

**Comment by tnj** [comment]
LGTM!

**Comment by henteko** [comment]
thanks!

**Pull request merged**